Minimum Cost For Tickets.cpp

```diff
@@ -0,0 +1,59 @@
+//time: O(365) ~ O(1)
```

Comment:
You cannot complexity of every question in this way. Check complexity for this and the above question